src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -1328,9 +1371,11 @@ void MessageHelper::SendOnAppUnregNotificationToHMI(
       hmi_apis::FunctionID::BasicCommunication_OnAppUnregistered;
 
   message[strings::params][strings::message_type] = MessageType::kNotification;
-  // we put hmi_app_id because applicaton list does not contain application on
```

Comment:
@IGapchuk unrelated changes

Reply:
@AKalinich-Luxoft revert in 36b0eadff97e5756477026c3d51f33e575f5d77a.